frontend/pages/topics/index.jsx

```diff
@@ -0,0 +1,49 @@
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import axios from "axios";
+import Layout from "../../components/Layout";
+
+
+export default function Topics({ topicsData }) {
+  const topics = topicsData["results"] || [];
+
+  //Style for the anchor used in topicNames
+  const topicAnchorStyle = {
+    color: 'black',
+  }
+  const topicNames = topics.map((topic) => (
+    <Grid item key={topic['key']} xs={6} sm={6} md={6}>
+      <a style={topicAnchorStyle} href={'./search?content_types=occurrences.occurrence&content_types=quotes.quote&content_types=images.image&content_types=sources.source&query=' + topic['key']}><u><strong>{topic['key']}</strong></u></a>
```

Comment:
By default, all content types are selected in search results, so we should be able to drop the `content_types` parameters and get the same results. Also, rather than searching for the topic with the query string, we should use the `topics` paramater. E.g.: ``href={`/search?topics=${topic["pk"]}`}``. Finally, by setting this in a [`next/link`](https://nextjs.org/docs/api-reference/next/link) component we can utilize client-side routing instead of reloading the application. E.g.: 
```javascript
import Link from "next/link";
<Link href={`/search?topics=${topic["pk"]}`}>
  <a {...propsExceptHref}>...</a>
</Link>
```

Reply:
Queries have been updated and Link component has been added.